src/common/registry/main/abstract-registry.ts

```diff
@@ -110,7 +110,8 @@ export abstract class AbstractRegistryService {
 
       // skip updating keys from 0 to `usedSigningKeys` of previous collected data
       // since the contract guarantees that these keys cannot be changed
-      const unchangedKeysMaxIndex = isSameOperator ? prevOperator.usedSigningKeys : 0;
+      const unchangedKeysMaxIndex =
+        isSameOperator && prevOperator.finalizedUsedSigningKeys ? prevOperator.finalizedUsedSigningKeys : 0;
```

Comment:
do we check prevOperator.finalizedUsedSigningKeys  on left side of '?' operator because of empty value of this field in current data in database ?

Reply:
yes
`isSameOperator && prevOperator.finalizedUsedSigningKeys` 
this condition makes checking this case

Reply:
can I resolve it?